Riot/Modules/Common/Recents/DataSources/RecentsDataSource.m

```diff
@@ -1121,7 +1121,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     {
         RecentsInvitesTableViewCell *tableViewCell = [tableView dequeueReusableCellWithIdentifier:[RecentsInvitesTableViewCell defaultReuseIdentifier]];
         
-        tableViewCell.invitesCount = self.recentsListService.invitedRoomListData.counts.total.numberOfRooms;
+        tableViewCell.invitesCount = self.invitesCellDataArray.count;
```

Comment:
Please check if this works with pagination (if any) otherwise we may need a different workaround